app/src/main/java/de/westnordost/streetcomplete/data/quest/DayNightCycle.kt

```diff
@@ -0,0 +1,7 @@
+package de.westnordost.streetcomplete.data.quest
+
+sealed class DayNightCycle
+
+object DayAndNight : DayNightCycle()
+object OnlyDay : DayNightCycle()
+object OnlyNight : DayNightCycle()
```

Comment:
Any reason to use a sealed class? An enum would do, too

```suggestion
enum class DayNightCycle { DAY_AND_NIGHT, ONLY_DAY, ONLY_NIGHT }
```